src/components/statsBar/StatsView.js

```diff
@@ -65,17 +65,25 @@ const styles = () => ({
     height: props.statsStyling.global.height ? props.statsStyling.global.height : '47px',
     margin: '0 auto',
   }),
-  statTitle: (props) => ({
+  statTitle: (props) => (props.statsStyling.global.horizontalStyle === true ? {
```

Comment:
Does it throw an error if the global.horizontal style is not provided in custodian?